spec/models/manageiq/providers/openshift/container_manager/refresher_spec.rb

```diff
@@ -144,16 +152,19 @@ def normal_refresh
 
     it "saves the objects in the DB" do
       expect(ContainerProject.count).to eq(8)
-      expect(ContainerImage.count).to eq(44)
-      expect(ContainerRoute.count).to eq(6)
-      expect(ContainerTemplate.count).to eq(30)
-      expect(ContainerReplicator.count).to eq(10)
+      expect(ContainerImage.count).to eq(39)
+      expect(ContainerRoute.count).to eq(5)
+      expect(ContainerTemplate.count).to eq(33)
+      expect(ContainerReplicator.count).to eq(6)
       expect(ContainerBuild.count).to eq(3)
       expect(ContainerBuildPod.count).to eq(3)
-      expect(CustomAttribute.count).to eq(532)
-      expect(ContainerTemplateParameter.count).to eq(264)
+      expect(CustomAttribute.count).to eq(564)
+      expect(ContainerTemplateParameter.count).to eq(367)
       expect(ContainerRoute.find_by(:name => "my-route-2").labels.count).to eq(1)
       expect(ContainerTemplate.find_by(:name => "my-template-2").container_template_parameters.count).to eq(1)
+      ContainerGroup.where.not(:container_build_pod_id => nil).each do |pod|
```

Comment:
if for some reason this `where` returned nothing, test would check 0 equalities and still be happy.
you want to check this for 3 pods right?  can you match them by name?
alternatively, start from ContainerBuildPod (which we assert above there are 3) and check their `.container_group.container_project.name`